google-cloud-spanner/src/main/java/com/google/cloud/spanner/connection/ConnectionStatementExecutorImpl.java

```diff
@@ -215,13 +226,17 @@ public StatementResult statementShowCommitResponse() {
     ResultSet resultSet =
         ResultSets.forRows(
             Type.struct(
-                StructField.of("COMMIT_TIMESTAMP", Type.timestamp()),
-                StructField.of("MUTATION_COUNT", Type.int64())),
-            Arrays.asList(
+                StructField.of(
+                    String.format("%sCOMMIT_TIMESTAMP", getNamespace(connection.getDialect())),
+                    Type.timestamp()),
+                StructField.of(
+                    String.format("%sMUTATION_COUNT", getNamespace(connection.getDialect())),
+                    Type.int64())),
+            Collections.singletonList(
                 Struct.newBuilder()
-                    .set("COMMIT_TIMESTAMP")
+                    .set(String.format("%sCOMMIT_TIMESTAMP", getNamespace(connection.getDialect())))
                     .to(response == null ? null : response.getCommitTimestamp())
-                    .set("MUTATION_COUNT")
+                    .set(String.format("%sMUTATION_COUNT", getNamespace(connection.getDialect())))
```

Comment:
Is this in the documentation of variables? If not could you add it to our document?

Reply:
I've added it to both the documents. This variable will only contain a value if `SET RETURN_COMMIT_STATS=true` has been executed first, and a read/write transaction has been committed.